guide/naming.md

```diff
@@ -0,0 +1,58 @@
+# VDD Naming Convetions / Semantic Versioning
+
+In order to facilitate tracability over several iterations of English
+and TLA+ specifications, and to make clear to external readers what is
+the state of a given document, we use the following convention:
+
+## English Spec
+
+As part of the version, each English specification should have one
```

Comment:
This would help address the point on which I was unclear re: where this stage is actually supposed to go.

```suggestion
As part of the version, the file name and title of each English specification should include reference to one
```